src/Printer/FormatPerservingPrinter.php

```diff
@@ -22,17 +22,18 @@ public function __construct(BetterStandardPrinter $betterStandardPrinter)
      * @param Node[] $newStmts
      * @param Node[] $oldStmts
      * @param Node[] $oldTokens
+     * @return mixed
      */
-    public function printToFile(SplFileInfo $fileInfo, array $newStmts, array $oldStmts, array $oldTokens): void
+    public function printToFile(SplFileInfo $fileInfo, array $newStmts, array $oldStmts, array $oldTokens)
```

Comment:
Could it return `bool` in case of change? It's easier to work with than `mixed`

Reply:
As I said above: as the `file_put_contents` return `int`, should we force `bool`?